docs/network-flow-visibility.md

```diff
@@ -352,17 +361,49 @@ commands:
 ./infra/vagrant/push_antrea.sh --flow-collector <externalFlowCollectorAddress>
 ```
 
-## ELK Flow Collector
+## Flow Collector
+
+Here we list two choices the external configured flow collector: IPFIX Collector and ELK Flow Collector. For each collector, we introduce how to deploy it and how to output or visualize the collected flow records information.
+
+### IPFIX Collector
+
+#### Deployment Steps
+
+The go-ipfix collector can be built from [go-ipfix library](https://github.com/vmware/go-ipfix). It is used to collect, decode and log the IPFIX records.
```

Comment:
again it seems that we use different terms for the same thing (go-ipfix collector, IPFIX collector)

Reply:
Thank you for catching that! Fixed.